### PR TITLE
add doc about using verified_at with IAL1 (LG-3464)

### PR DESCRIPTION
**Pull request by mitchellhenke**

### CODE REVIEW

File: _pages/attributes.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ When the user's identity was last verified (or empty if it has never been verifi
 
 Seconds since the Unix Epoc
 
-Requires the `profile` or `profile:verified_at` scope.
+Requires the `profile` or `profile:verified_at` scope. To get the attribute with IAL1, the `profile:verified_at` scope must be requested.
```
Comment (zachmargolis) [inline _pages/attributes.md:176]: Maybe we should just simplify to

```suggestion
Requires the `profile:verified_at` scope.
```

and we updated https://developers.login.gov/attributes/ as well?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _pages/attributes.md (hunk 1 above)
Comment (mitchellhenke) [reply _pages/attributes.md:176]: I think it's ok to simplify, but did you mean to link to a different URL? This change is updating the page you linked 🙂

Comment (zachmargolis) [reply _pages/attributes.md:176]: whoooops I meant the OIDC page, the description of `verified_at` towards the bottom

https://developers.login.gov/oidc/

Comment (mitchellhenke) [reply _pages/attributes.md]: Changed OIDC page to use the same language.

Comment (zachmargolis) [reply _pages/attributes.md]: sweeeeeeet thanks

**Pull request merged**